README.md

```diff
@@ -12,7 +15,18 @@ See [Beaker Installation](docs/Beaker-Installation.md).
 
 Documentation for Beaker can be found in this repository in [the docs/ folder](docs/README.md).
 
-The [Getting Started](docs/How-To-Beaker.md) is a great place to start for new Beaker users.
+A high-level overview of how our docs are organized will help you know where to look for certain things:
```

Comment:
suggest rewording.  maybe just:

`#Table of Contents`

also in docs/README.md